packages/babel-preset-env/test/fixtures/preset-options-add-used-built-ins/source-type-script/options.json

```diff
@@ -2,9 +2,7 @@
   "presets": [
     ["../../../../lib", {
       "modules": false,
-      "targets": {
-        "node": "4.0.0"
-      },
+      "targets": "node 4.0.0",
```

Comment:
Instead of modifying existing tests, should we just add new ones to make sure both syntaxes work (for now)?